crates/hyperswitch_connectors/src/connectors/fiserv.rs

```diff
@@ -179,53 +188,37 @@ impl ConnectorValidation for Fiserv {
         match capture_method {
             enums::CaptureMethod::Automatic | enums::CaptureMethod::Manual => Ok(()),
             enums::CaptureMethod::ManualMultiple | enums::CaptureMethod::Scheduled => Err(
-                connector_utils::construct_not_implemented_error_report(capture_method, self.id()),
+                utils::construct_not_implemented_error_report(capture_method, self.id()),
             ),
         }
     }
 }
 
 impl api::ConnectorAccessToken for Fiserv {}
 
-impl ConnectorIntegration<api::AccessTokenAuth, types::AccessTokenRequestData, types::AccessToken>
-    for Fiserv
-{
+impl ConnectorIntegration<AccessTokenAuth, AccessTokenRequestData, AccessToken> for Fiserv {
     // Not Implemented (R)
 }
 
 impl api::Payment for Fiserv {}
 
 impl api::PaymentToken for Fiserv {}
 
-impl
-    ConnectorIntegration<
-        api::PaymentMethodToken,
-        types::PaymentMethodTokenizationData,
-        types::PaymentsResponseData,
-    > for Fiserv
+impl ConnectorIntegration<PaymentMethodToken, PaymentMethodTokenizationData, PaymentsResponseData>
+    for Fiserv
 {
     // Not Implemented (R)
 }
 
 impl api::MandateSetup for Fiserv {}
 
 #[allow(dead_code)]
```

Comment:
Do we need these `#[allow(dead_code)]` annotations here (and elsewhere in the file)?

Reply:
I am not changing any code logic while moving connectors for now. This is being to avoid rigorous testing for these PR's. 
We can address all these issues when these connectors go for an audit.